synapse/telepath.py

```diff
@@ -28,7 +28,7 @@
 
 aha_clients = {}
 
-async def addAhaUrl(url):
+async def addAhaUrl(url, anit=True):
```

Comment:
Do you think it would be safe to eliminate the anit option and just have it behave as the False case would?

Reply:
I think it would probably be okay to do that, `_initAhaRegistry` in `cell.py` assumes that `addAhaUrl` returns a telepath client, but that would be easy enough to clean up. I don't think anything else would break.

Reply:
anit option eliminated